app/src/main/java/de/othr/plantico/ui/homescreen/WateringAdapter.kt

```diff
@@ -56,15 +80,40 @@ class WateringAdapter : ListAdapter<Plant, WateringAdapter.WateringViewHolder>(P
         }
     }
 
-    class PlantsComparator : DiffUtil.ItemCallback<Plant>() {
-        override fun areItemsTheSame(oldItem: Plant, newItem: Plant): Boolean {
+    class OwnedPlantsComparator : DiffUtil.ItemCallback<OwnedPlant>() {
+        override fun areItemsTheSame(oldItem: OwnedPlant, newItem: OwnedPlant): Boolean {
             return oldItem === newItem
         }
 
-        override fun areContentsTheSame(oldItem: Plant, newItem: Plant): Boolean {
+        override fun areContentsTheSame(oldItem: OwnedPlant, newItem: OwnedPlant): Boolean {
             return oldItem.plantName == newItem.plantName
         }
     }
 
 
 }
+
+fun Date.toPlanticoString():String{
+    val suffixes = arrayOf(
+        "th", "st", "nd", "rd", "th", "th", "th", "th",
+        "th", "th","th", "th", "th", "th", "th", "th", "th", "th", "th", "th",
+        "th", "st", "nd", "rd", "th", "th", "th", "th", "th", "th", "th", "st")
+    val c = Calendar.getInstance()
+    c.time = this
+    val day = c[Calendar.DAY_OF_MONTH]
+    val month_name = SimpleDateFormat("MMMM", Locale.ENGLISH).format(c.getTime())
+    return ""+day + suffixes[day] + " " + month_name
+
+}
+
+fun Date.addDays(days: Int):Date{
+    val c: Calendar = Calendar.getInstance()
+    c.time = this
+    c.add(Calendar.DATE, days)
+    return c.time
+}
+
+fun Date.nowUTC():Date{
+    return Date.from(LocalDateTime.now().toInstant(ZoneOffset.UTC))
+
+}
```

Comment:
Yeah. The thing is. Every class can already use these. You only have to import them. It would be clever to create maybe a util File where all these Extensions and Conversions are stored separately.